extensions/amp-consent/amp-consent.md

```diff
@@ -180,20 +179,20 @@ Take the following config as an example:
 ```json
 {
   "onUpdateHref": "https://example.com/update-consent",
-  "useCache": false,
-  
+  "promptUI": "consent-ui",
+  "promptIfUnknown": false,
+
   "geoOverride": {
     "geo1": {
-      "useCache": true,
-      "promptUI": "consent-ui",
+      "promptIfUnknown": true,
     }, 
     "geo2": {
       "checkConsentHref": "https://example.com/check-consent",
+      "promptIfUnknown": "waitForCheckConsent",
     },
     "unknown": {
-      "useCache": true,
-      "promptUI": "consent-ui",
       "checkConsentHref": "https://example.com/check-consent",
+      "promptIfUnknown": true,
```

Comment:
given there's no use case, I'd propose that we don't require the `unknown` setting